### ISSUE TITLE
Reagarding discripency in tensorboard validation accuracy and the one printed during training 

### ISSUE THREAD

**Issue by ankit1-iitbhu** [tensorflow/tensorboard#4667]
When I am running a random search for hyperparameter tunning. The accuracies appearing after each training on print screen for training and validation sets are different which is not appearing in tensorboard for the same run. In tensorboard the validation accuracy always appears lesser than the print value.

![image](https://user-images.githubusercontent.com/37747800/107321514-ab027300-6ac8-11eb-963b-1f04910581ec.png)
![image](https://user-images.githubusercontent.com/37747800/107321520-b3f34480-6ac8-11eb-8b7f-cb06ee342d40.png)

Further, the number of layer count is different from layers shown for most of the model.

**Comment by arghyaganguly** [comment]
@ankit1-iitbhu , please provide a colab link with the code or an alternate file with the full code to reproduce this issue.
Also please provide the log directory.

**Comment by arghyaganguly** [comment]
Closing this as it has been  awaiting response for a prolonged period.Please fell free to reopen with the requested information.Thanks.
